Triaging-process.md

```diff
@@ -0,0 +1,57 @@
+## General
+This wiki outlines the steps to triage new and existing issues. This ensures that all issues are being effectively tracked and prioritized.
+
+### Steps to triage incoming issues coming into Oppia as a whole (first-stage triage):
+The Dev Workflow team lead (currently Chris) should do the following every week for any [filed issues that aren’t assigned to a project](https://github.com/oppia/oppia/issues?q=is%3Aissue+is%3Aopen+no%3Aproject):
+
+- Check that the issue is written clearly enough. Request clarification if needed.
+- Assign the issue to the appropriate task force, if applicable. Otherwise, assign it to the relevant team (LaCE, CD, Dev Workflow) based on which type of user it affects.
+- If you’re not sure how to categorise an issue, feel free to ping Sean for help!
+
+### Steps to triage incoming issues for each team (second-stage triage):
+Team leads should do an audit every week to categorise new issues in their GitHub Projects board, following these steps:
+
+1. Verify that the issue lies within the scope of the team (see definitions below).
+    - If not, assign the issue to the appropriate project/team.
+2. Determine whether the issue is a bug or a enhancement
+    - If the issue is a feature request (enhancement), contact the relevant PM (product manager) for the team and ask them to take a look and decide whether it needs to be worked upon. (See the list of PMs [below](##pm-contacts-for-each-team).)
```

Comment:
I don't think the `[below](##pm-contacts-for-each-team)` link is working please fix it.

Reply:
Fixed